.cloudbuild/staging/appengine/http.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+)
+
+var (
+	intlMatcher = regexp.MustCompile(`^/intl/([-_\w]+)(/.*|)$`)
+)
+
+func main() {
+	fileServer := http.FileServer(http.Dir("prod"))
+	handler := rewriteLang(fileServer)
+
+	http.Handle("/", handler)
+
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
+	addr := ":" + port
+	log.Printf("Serving on %s...", addr)
+	http.ListenAndServe(addr, nil)
+}
+
+// // stripPrefix removes a matching prefix from the given URL, but also inserts a
```

Comment:
I would generally opt for deleting commented out functions :)

Reply:
good call. The internal elfbuilder version of this used this part for whitelist code (which I've removed).
But turns out if everything is public in the repo, then the fact that we stage a test version of Santa is not very secret either...